examples/cluster-dns/images/frontend/client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
I openned https://github.com/GoogleCloudPlatform/kubernetes/pull/8129 to catch if people just use the boilerplate.*.txt without editting the date...